litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -104,10 +104,14 @@ rules:
     resources:
       - pods
       - pods/exec
+      - services
+      - nodes
     verbs:
       - create
       - patch
       - update
+      - get
```

Comment:
is it required for namespace manifest too?

Reply:
Not required in namespace manifest. In namespace scope, these permission are required to get external IP and port, for namespace scope we will use PortalEndpoint as an env